text/0000-Component-Manager-Bounds.md

```diff
@@ -0,0 +1,177 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Component Manager Bounds
+
+## Summary
+
+This is a follow up custom components RFC that outlines how we expose a component's DOM structure and associated semantics.
+
+## Motivation
+Historically, classic Ember components have had a built in escape hatch for doing DOM manipulation via `didInsertElement`. This allowed for components to encapsulate some 3rd party JavaScript library or do some fine grain DOM manipulation. We feel that is ability to drop down to a lower level abstraction makes components powerful.
+
+Today, the custom components are not given direct access to the DOM structure; internally known as `Bounds`. This limitation means that custom components cannot easily encapsulate functionality in the same way that classic Ember components can. Expanding the capabilities of custom components to expose the `Bounds` is in order to increase the usefulness of the abstraction.
+
+## Detailed design
+
+In [RFC#2013](https://github.com/emberjs/rfcs/blob/master/text/0213-custom-components.md#detailed-design) we outlined the component lifecycle and how a custom component is associated with a manager. This design builds off off of that existing work and concepts.
+
+### `elementHook` capability
```

Comment:
`elementHook` doesn't really seem to fit with the other capability names. Suggestions:

* `elementAccess`
* `domAccess`
* `domAccessCallbacks`
* `domCallbacks`
* `elementCallbacks`
* `bounds` (I like this one best 😄)

Thoughts?